bin/eks-create-cluster.sh

```diff
@@ -55,7 +55,7 @@ do
       echo "EKS cluster is ready"
       aws eks update-kubeconfig --name $EKS_CLUSTER_NAME --kubeconfig ~/.kube/config-eks
       export KUBECONFIG=~/.kube/config-eks
-      kubectx ${CLUSTER_ARN}
+      kubectl config set-context ${CLUSTER_ARN}
```

Comment:
Why? Is kubectx not supported on EKS?

Reply:
We ran into some issues when working with multiple kubernetes environments and this gave us more consistent results

Reply:
Could you please share the problems you hit with the team? Because we are recommending the use of kubectx in docs and using it in our scripts, and if there are problems, perhaps we should move away from it. Thanks.

Reply:
@wajihahmed was the one experiencing most of the inconsistencies with kubectx so he may be able to fill you in better. I had a few instances where I had not cleared my .kube/config-eks after creating a lot of new environments and it would occasionally not switch the context correctly in the script.

Reply:
> Why? Is kubectx not supported on EKS?

Kubectx is a third party tool.  It is not part of official kubernetes

Reply:
True, but it is part of CDM - we're telling people to install it and use it, and we're using it in our own scripts.

Reply:
Yes but AFAIK it is not part of any scripts in forgeops.  We can certainly mention it as a convenient tool to switch contexts when developing.  However in the scripts, I want to strictly stick to kubectl wherever possible.

Reply:
Ah, you're right. I figured since it was in eks-create-cluster that it had been copied in from gke-create-cluster.
So, would you suggest I pull it out of the docs altogether? It sure is nice for users but if it's flaky on EKS...